ui/components/src/StatusHistoryChart/StatusHistoryChart.tsx

```diff
@@ -153,7 +147,7 @@ export function StatusHistoryChart(props: StatusHistoryChartProps) {
   };
 
   return (
-    <Box style={{ height: height }} sx={{ overflow: 'auto' }}>
+    <Box sx={{ height: height, overflow: 'auto' }}>
```

Comment:
We do not but high dynamic value in sx props: https://mui.com/system/getting-started/the-sx-prop/#dynamic-values. This change need to be reverted

Reply:
Thank you! Brought old code by mistake.